src/hapi/pipelines/utilities/sector.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from typing import Dict
+
+from hdx.scraper.base_scraper import BaseScraper
+from hdx.utilities.dateparse import parse_date
+from sqlalchemy.orm import Session
+
+from hapi.pipelines.database.db_sector import DBSector
+
+logger = logging.getLogger(__name__)
+
+
+class Sector(BaseScraper):
+    def __init__(self, session: Session, datasetinfo: Dict[str, str]):
+        super().__init__(
+            "sector",
+            datasetinfo,
+            dict(),
+        )
+        self._session = session
+        self.data = {}
+        self._scraped_data = []
```

Comment:
@turnerm this feels more clear to me - does it to you?  I can still switch to a data class if that would help even more.

Reply:
This improves it a lot, you can keep as is!